Sources/Datadog/RUM/Instrumentation/RUMInstrumentation.swift

```diff
@@ -46,10 +25,9 @@ internal final class RUMInstrumentation: RUMCommandPublisher {
         }
     }
 
+    let viewsInstrumentation: RUMViewsHandler
     /// RUM Views auto instrumentation, `nil` if not enabled.
-    let viewsAutoInstrumentation: ViewsAutoInstrumentation?
-    /// `SwiftUI.View` RUM instrumentation
-    let swiftUIViewInstrumentation: SwiftUIViewHandler
+    let viewsAutoInstrumentation: UIViewControllerSwizzler?
```

Comment:
I'm not very satisfied with the naming here

Reply:
I'm not sure yet if `UIKitRUMUserActionsHandler` will support all SwiftUI actions. But I agree with clarifying by using `uiViewControllerSwizzler`, will see if I need to revisit in RUMM-1658.